integreat_cms/cms/templates/hix_widget.html

```diff
@@ -11,7 +11,7 @@
 {% block collapsible_box_content %}
     <div class="relative">
         <div id="hix-loading"
-             class="hidden absolute w-full h-full bg-white bg-opacity-75">
+             class="absolute w-full h-full bg-white bg-opacity-75">
             <div class="relative top-1/3 m-auto w-1/3">
```

Comment:
```suggestion
            <div class="relative top-1/3 text-center">
```
Not your fault, but I noticed the text is not really centered (only when the loading text has exactly the width of 1/3rd of the widget by chance)